gpu-service/jobs.py

```
@@ -0,0 +1,486 @@
+            return {"status": "success", "result": f"Upserted {upserted} faces"}
+        return {"status": "success", "message": "No faces detected"}
+
+    except Exception as e:
+        logger.error(f"Error processing image {object_name}: {str(e)}")
+        return {"status": "error", "error": str(e)}
+    finally:
+        if 'image_response' in locals():
+            image_response.close()
+            image_response.release_conn()
+
+def process_video_upload(bucket_name, object_name, event_data=None):
```

Ruff (0.11.9)

`process_video_upload` is too complex (13 > 10)

(C901)